azure-pipelines.yml

```diff
@@ -35,11 +35,9 @@ stages:
         sudo modprobe overlay
         CACHE_OPTIONS="SONIC_DPKG_CACHE_METHOD=rcache SONIC_DPKG_CACHE_SOURCE=/nfs/dpkg_cache/broadcom"
         ENABLE_DOCKER_BASE_PULL=y make configure PLATFORM=broadcom
-        make USERNAME=admin SONIC_BUILD_JOBS=$(nproc) $CACHE_OPTIONS target/sonic-broadcom.bin
+        trap "sudo rm -rf fsroot" EXIT
+        false && make USERNAME=admin SONIC_BUILD_JOBS=$(nproc) $CACHE_OPTIONS target/sonic-broadcom.bin
```

Comment:
>false [](start = 8, length = 5)

The `false` will fail the whole line immediately and the `make` will be skipped, right?